Bootstrap v4 - (Buttons) Button - Primary.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Button - Primary\u00a0",
```

Comment:
The NON-BREAKING SPACE are not good.. I think I need to add a fix to insure on export it strips those.
`\u00a0`